src/howTos/dev/services.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Develop a service from your application
+summary: Execute server-side and asynchronous code from your application.
+---
+
+
+## What is it for?
+
+Applications may require server-side code execution that could not and should not be in the client. This might be useful for heavy computations or for tasks triggered after some events, typically after data is retrieved through [konnectors](https://cozy.github.io/cozy-stack/konnectors.html) or mobile/desktop sync, without the user being on the application.
+
+In contrast to konnectors, services have the same permissions as the web application and are not intended to collect information from the outside. It is rather meant to asynchronously analyse data inside the cozy and emit some output once the task is done. However, they share the same mechanisms as the konnectors to describe how and when they should be executed: via our [trigger system](https://github.com/cozy/cozy-stack/blob/master/docs/jobs.md).
+
+
+ background and offline process to analyse the user's data and emit some output even without the user being on the application. These part of the application are called services and can be declared as part of the application in its manifest.
```

Comment:
Maybe something is missing at the beginning here?

Reply:
Indeed, looks like a failed copy-paste :)